test/lib/ufe/testPayloadCommands.py

```diff
@@ -144,5 +144,30 @@ def testLoadAndUnloadPayloadCommands(self):
         cmd.redo()
         self.assertTrue(prim.IsLoaded())
 
+    @unittest.skipUnless(mayaUtils.mayaMajorVersion() > 2022, 'Delete restriction on delete was only after Maya 2022.')
```

Comment:
Minor, but I think its clearer to use ">= 2023" and say "Delete restriction requires Maya 2023 or greater". I think most of the tests do that.